cypress/helpers/repository.js

```diff
@@ -24,7 +27,7 @@ export function pushLocalChanges(numberOfChanges) {
     // The time to wait depends on the number of changes
     menu
       .changesNumber({
-        timeout: 5000 * (numberOfChanges + 1),
+        timeout: 2 * 60 * 1000 * (numberOfChanges + 1),
```

Comment:
Do we still need this with the new transactional push?

Reply:
I think so, depends on how the screenshots are handled

Reply:
Looks like it: https://github.com/prismicio/slice-machine/actions/runs/4287746526/jobs/7468979692#step:8:819
pushing screen shots takes a while longer